### DISCUSSION TITLE
[Question] How to use API responses from Flask in Flet Static Website?

### DISCUSSION THREAD

**Discussion by gamartin23** [flet-dev/flet#2891] [Q&A]
### Question

Hi! I've been scouring the internet for examples of static websites dealing with this trouble and i could not find any made with Flet. 
All i need is to see an example of a Flet website that does GET and POST requests with a button, changing a value on screen, and does work when built into a static website. I'd love to see what libraries or methods i need to use, because this is a bit above my paygrade and i'm getting way too stressed for a passion project 😅
thanks!

code sample is an example of the kind of functions i need to make work within this page

edit: yes, esentially i ask for an example of how to replace the library Requests.

### Code sample

```python
async def get_api():
        api = 'api.example.com/users'
        async with aiohttp.ClientSession() as session:
            async with session.get(api) as response:
                if response.status == 200:
                    data = await response.json()
                    await session.close()
                    return data["count"]
                else:
                    await session.close()
                    return 'Servicio no disponible'
    count =  await get_api()
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
> esentially i ask for an example of how to replace the library Requests.

You can give [`Pyodide.http`](https://pyodide.org/en/stable/usage/api/python-api/http.html) a try.
